DaiNekoIchis_TomeOfSpells/cards/WhiteWind.ts

```diff
@@ -0,0 +1,139 @@
+            },
+            "noRotation": true,
+            "rotationSpeed": {
+                "min": 0,
+                "max": 0
+            },
+            "lifetime": {
+                "min": 0.2,
+                "max": 0.8
+            },
+            "blendMode": "add",
+            "frequency": 0.001,
+            "emitterLifetime": waitTime / 2.0,
+            "maxParticles": 500 * radius,
+            "pos": {
+                "x": 0,
+                "y": 0
+            },
+            "addAtBack": false,
+            "spawnType": "ring",
+            "spawnCircle": {
+                "x": 0,
+                "y": 0,
+                "r": radius,
+                "minR": 0
+            }
+        }, [texture]);
+        simpleEmitter(position, particleConfig);
+}
+
+const cardId = 'Healing Breeze';
+const baseRange = 100;
+const waitTime = 2;
+const spell: Spell = {
+    card: {
+        id: cardId,
+        category: CardCategory.Blessings,
+        supportQuantity: true,
+        manaCost: 50,
+        healthCost: 0,
+        expenseScaling: 2,
+        probability: probabilityMap[CardRarity.RARE],
+        allowNonUnitTarget: true,
+        thumbnail: 'spellmasons-mods/DaiNekoIchis_TomeOfSpells/graphics/WhiteWind.png',
+        sfx: 'heal',
+        description: [`Heals targets in an area around self equal to own health.`],
+        effect: async (state, card, quantity, underworld, prediction) => {
+            let emitter;
+            let adjustedRange = baseRange * (1 + ((quantity - 1) * 0.5) + (0.25 * state.aggregator.radiusBoost));
+            if (prediction) {
+                drawUICirclePrediction(state.casterUnit, adjustedRange, colors.targetingSpellGreen, 'Target Radius');
+            } else {
+                makeWhiteWindParticles(state.casterUnit, adjustedRange, underworld, prediction);
+            }
+            if (!prediction && !globalThis.headless) {
+                await new Promise((resolve) => {
+                    setTimeout(resolve, waitTime * 1000);
+                });
+            }
+            let entities = underworld.getEntitiesWithinDistanceOfTarget(state.casterUnit, adjustedRange, prediction);
+            for (let entity of entities) {
+                if (Unit.isUnit(entity)) {
+                    let target: IUnit = entity;
+                    Unit.takeDamage({unit: target, amount: -state.casterUnit.health}, underworld, prediction);
+                }
+                playDefaultSpellSFX(card, prediction);
+            }
+            if (!prediction && !globalThis.headless) {
+                for (let follower of underworld.particleFollowers) {
+                    if (follower.emitter.name === WHITE_WIND_EMITTER_NAME && follower.target == state.casterUnit) {
+                        // Remove emitter
+                        ParticleCollection.stopAndDestroyForeverEmitter(follower.emitter);
+                        break;
+                    }
+                }
+            }
```

Comment:
I think this part is unnecessary, since this is just a regular emitter and isn't added to the particleFollowers in the first place.  I'll double check when I QA.